src/components/dashboard/dashboardItem/LoginTable.js

```diff
@@ -0,0 +1,84 @@
+/**
+ *
+ * @author mian
+ *
+ * A table of recent logins for the user.
+ *
+ */
+
+import React from "react";
+import { useQuery } from "react-query";
+
+import { logins, LOGINS_QUERY_KEY } from "serviceFacades/users";
+import ErrorTypographyWithDialog from "components/error/ErrorTypographyWithDialog";
+import { useTranslation } from "i18n";
+import {
+    formatDate,
+    getFormattedDistance,
+} from "components/utils/DateFormatter";
+import {
+    useTheme,
+    Skeleton,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Paper,
+} from "@mui/material";
+
+export default function LoginsTable() {
+    const { t } = useTranslation("dashboard");
+    const { t: i18nCommon } = useTranslation("common");
+
+    const theme = useTheme();
+    const { status, data, error } = useQuery({
+        queryKey: [LOGINS_QUERY_KEY],
+        queryFn: () => logins({ limit: 5 }),
+    });
+
+    if (status === "error") {
+        return (
+            <div style={{ padding: theme.spacing(1) }}>
+                <ErrorTypographyWithDialog
+                    errorObject={error}
+                    errorMessage={t("loginsTableError")}
+                />
+            </div>
+        );
+    }
+    if (status === "loading") {
+        return <Skeleton variant="rectangular" height={200} />;
+    }
+    return (
+        <TableContainer component={Paper}>
+            <Table size="small">
+                <TableHead>
+                    <TableRow>
+                        <TableCell>{t("loginTime")}</TableCell>
+                        <TableCell></TableCell>
+                        <TableCell>{t("ipAddress")}</TableCell>
+                    </TableRow>
+                </TableHead>
+                <TableBody>
+                    {data["logins"].map((row) => (
+                        <TableRow>
+                            <TableCell>
+                                {i18nCommon("timestamp", {
+                                    timestamp: getFormattedDistance(
+                                        row["login_time"] / 1000
+                                    ),
+                                })}
+                            </TableCell>
+                            <TableCell>
+                                ({formatDate(row["login_time"])})
+                            </TableCell>
```

Comment:
Personally I think it makes more sense to swap these columns, so something like: `2025-01-21 09:10:11    (about 1 day ago)`

Reply:
I suppose with that I could also remove the second-column thing, because the dates will be the same width regardless. so that definitely seems cleaner. Trying too hard to copy other sites' stuff I think, heh

Reply:
![image](https://github.com/user-attachments/assets/866b34b1-dfb4-4e0e-b8bc-fbde9e64f99c)

here's how I've got it now, and I think this looks pretty good -- added the italics as well, to visually distinguish the parenthetical a little bit more